data-raw/babies.R

```diff
@@ -0,0 +1,28 @@
+## code to prepare `babies` dataset goes here
+
+babies <- readr::read_csv("https://wjhopper.github.io/SDS-201/data/babies.csv")
+
+usethis::use_data(babies, overwrite = TRUE)
+library(tidyverse)
+
+# Creating new date variable
+
+babies$birthday <- babies$date
+
+# Creating new dates
```

Comment:
Look into `lubridate::parse_date_time()` to extract the date in the right format

Reply:
What is the right format?

Reply:
Dates should have type `dttm` or `date`